docs/src/protractor-js.md

```diff
@@ -117,7 +117,22 @@ Here's how to polyfill `waitForAngular` function in Playwright Test:
     ​await executeScriptAsync(page, clientSideScripts.waitForAngular, '');
     }
     ```
-
+   If you don't want to keep a version protractor around, you can also use this simpler approach using this function:
+   ```js
+   async function waitForAngular(page) {
+     await page.evaluate(async () => {
+       // @ts-expect-error
+       if (window.getAllAngularTestabilities) {
```

Comment:
do I understand correctly that this works only with Angular2?

Reply:
Yes, Angular 2+. I personally haven't seen an AngularJS project for a long time now. LTS for AngularJS runs until December 31, 2021, so I'd personally be happy not to include it.

Reply:
Awesome, let's clarify this in the docs, and it'll be good to go!

Reply:
Thanks! Just did that.